### DISCUSSION TITLE
[flutterfire_ui] FirestoreListView: How to provide widget builder for zero items

### DISCUSSION THREAD

**Discussion by satvikreddy** [firebase/flutterfire#7656] [Q&A] [answered]
FirestoreListView shows a blank screen if query returns no data.

Is there a way to provide a emptyWidgetBuilder to show some text like "no data" when query has no data?

**Comment by rrousselGit** [comment, marked as answer]
For any more advanced UI use-cass, you should switch to using FirestoreQueryBuilder
